bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/InfluxDBPersistenceService.java

```diff
@@ -157,26 +189,26 @@ public void store(Item item) {
 
     @Override
     public void store(Item item, @Nullable String alias) {
-        if (checkConnection()) {
-            InfluxPoint point = itemToStorePointCreator.convert(item, alias);
-            if (point != null) {
-                try {
-                    influxDBRepository.write(point);
-                    logger.trace("Stored item {} in InfluxDB point {}", item, point);
-                } catch (UnexpectedConditionException e) {
-                    logger.warn("Failed to store item {} in InfluxDB point {}", point, item);
-                }
+        if (!serviceActivated) {
+            logger.warn("InfluxDB service not ready. Storing {} rejected.", item);
```

Comment:
Rather item.getName()

Reply:
I have thought about that, too. But logging `item` also logs the state, not only the item name. That's why I kept `item`.

Reply:
Ok